style.css

```diff
@@ -0,0 +1,258 @@
+font-family {
+	font-family: "Source Sans Pro Bold";
+	@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;600&display=swap');
+}
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	font-size: 62.5%; /* this makes 1em = 10px */
+	background-color: #F3F1F9;
+	
+}
+
+.navigationBar {
+	display: grid ;
+	grid-template-columns: 25% 75%;
+	background-color: #270483;
+/*	background-color: #630480;*/
+	height: 60px;
+	position: fixed; /* Set the navbar to fixed position */
+	top: 0; /* Position the navbar at the top of the page */
+	width: 100%; /* Full width */
+	z-index: 10;
+
+}
+
+nav {
+	/*justify-self: end;*/
+	display: flex;
+	justify-content: space-evenly; 
+	
+}
+
+nav a {
+	align-self: center;
+	color: white;
+	font-family: 'Poppins', sans-serif;
+	text-decoration: none;
+	font-size: 2em;
+	padding: 8px;
+	border-radius: 5px;
+}
+nav a:hover{
+	background-color: #D40454;
+}
+
+.navigationBar div {
+	padding: 15px 0px 10px 20px;
+	font-family: 'Poppins', sans-serif;
+	font-size: 2em;
+	color: #F4E4FC;
+}
+.navigationBar img {
+	position: absolute;
+	top: 0;
+	left: 50px;
+}
+
+.container {
+	padding-top: 60px;
+	padding-left: 5%;
+	padding-right: 5%;
+	font-family: 'Poppins', sans-serif;
+	font-size: 2em;
+	color: #270483;
+}
+.aboutMe a{
+	font-family: "Source Sans Pro Bold", sans-serif;
+	text-decoration: none;
+	color: #270483;
+}
+
+video {
+	z-index: -1;
+    height: 100%;
+    width: 100%;
+}	
+
+.intro {
+	padding: 10% 5% 5% 5%;
+	font-family: 'Poppins', sans-serif;
+	z-index: 10;
+	position: absolute;
+	top: 10%;
+	left: 16px;
+	z-index:2;
+	color: white;
+
+}
+
+.intro h1 {
+	font-size: 5em;
+}
+.intro p {
+	font-size: 3em;
+	font-family: 'Source Sans Pro', sans-serif;
+}
+
+.aboutMe p {
+	font-size: 1em;
+	font-family: 'Source Sans Pro', sans-serif;
+	align-self: center;
+	padding: 5%;
+}
+
+.bigBlock {
```

Comment:
`bigBlock` is not a very descriptive/semantic name - it refers to what the div looks like rather than what it is. That said I'm not sure what name is good for it